src/osd/OpRequest.h

```diff
@@ -54,7 +54,11 @@ struct OpRequest : public TrackedOp {
   void _dump(ceph::Formatter *f) const override;
 
   bool has_feature(uint64_t f) const {
+#ifdef WITH_SEASTAR
+    ceph_abort("In crimson, conn is independently maintained outside Message");
```

Comment:
`get_connection()` already has the abort. By adding it here as well you want to make the story more visible, right?

Reply:
Right. In addition, because of `using ConnectionRef = void*;` we have a compile time checking, so it must be added here as well.